crates/rpc/rpc/src/eth/api/transactions.rs

```diff
@@ -480,8 +493,28 @@ where
     }
 
     async fn send_raw_transaction(&self, tx: Bytes) -> EthResult<B256> {
-        let recovered = recover_raw_transaction(tx)?;
+        #[cfg(feature = "optimism")]
+        if let Some(endpoint) = self.network().sequencer_endpoint() {
+            let body = serde_json::to_string(&serde_json::json!({
+                "jsonrpc": "2.0",
+                "method": "eth_sendRawTransaction",
+                "params": [format!("0x{}", hex::encode(tx.clone()))],
```

Comment:
this should be a separate function, needs docs

Reply:
addressed in https://github.com/anton-rs/op-reth/pull/132